lib/optimizely/event_builder.rb

```diff
@@ -50,8 +50,9 @@ class BaseEventBuilder
 
     attr_reader :config
 
-    def initialize(config)
+    def initialize(config, logger)
       @config = config
+      @logger = logger
```

Comment:
Please add logger on the top like config: attr_reader :config

Reply:
It's not necessary.